behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -368,18 +368,18 @@ public function assertVariationEditMode($expected_state) {
     if (!in_array($expected_state, array('active','inactive','hidden','disabled'))) {
```

Comment:
Yep. This is deleted along with the simple_ab feature, and to be pushed with the rest of Behat updates.